addons/spreadsheet/static/src/chart/odoo_chart/odoo_bar_chart.js

```diff
@@ -38,8 +44,9 @@ chartRegistry.add("odoo_bar", {
 function createOdooChartRuntime(chart, getters) {
     const background = chart.background || "#FFFFFF";
     const { datasets, labels } = chart.dataSource.getData();
+    const dataSetFormat = chart.dataSets && getChartDatasetFormat(getters, chart.dataSets);
```

Comment:
That seems wrong, `getChartDatasetFormat` tries to find the format based on the ranges of the chart .... But odoo charts are not based on ranges, so that'll do nothing. 

I'm pretty sure the format of odoo charts is always undefined no ?

Reply:
Ahh, correct, I completely overlooked this :man_facepalming:😅